File: 2. Java/1. Bronze/28702.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

class Main {
    public static void main(String[] args) throws IOException {
        // Variables
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String line;
        int number = 0;

        // Take Input
        for (int i = 0; i < 3; i++) {
            line = br.readLine();
            
            if ((line.charAt(0) == 'B') || (line.charAt(0) == 'F')) continue;
            else {
                number = Integer.parseInt(line) + (3 - i);
                break;
            }
        }

        // Print Result
        if (number % 3 == 0) {
            if (number % 5 == 0) System.out.println("FizzBuzz");
            else System.out.println("Fizz");
        }
        else {
            if (number % 5 == 0) System.out.println("Buzz");
            else System.out.println(number);
        }
    }
}

File: 3. Python/1. Bronze/28702.py
import sys

# Variables
line = []

# Take Inputs
for _ in range(3): line.append(sys.stdin.readline().strip())

# Find Number
for i in range(3):
    if (line[i].isdigit()):
        number = int(line[i]) + (3 - i)
    
# Print Result
if (number % 3 == 0):
    if (number % 5 == 0): print("FizzBuzz")
    else: print("Fizz")
else:
    if (number % 5 == 0): print("Buzz")
    else: print(number)